### PR TITLE
Use adsk-npm-license-puller (internal tool) to generate about box content

### PR DESCRIPTION
**Pull request by mjkkirschner**
This PR does the following:
* adds a dev dependency on `adsk-npm-license-puller` of the latest version (thanks @twastvedt for your work here)
* this tool will only exist when using adsk npm registry!
* adds a npm script `generate_license` which will create two different about box .html files.

Next:
- [x] send a similar PR to notification center
- [ ] in Dynamo - consume the about box html with a webBrowser control (or similar) in the about box.

### CODE REVIEW

**Comment by QilongTang** [comment]
@mjkkirschner Is the duplication intended?

**Comment by mjkkirschner** [comment]
@QilongTang yes, the script I added generates both the direct dependencies and transitive dependencies about boxes for ease of tracking and integration., and because it's not clear what is the best practice - for now I am planning to integrate the direct dependency about box content, but having the full list in the git history is very useful over time I think.

If possible it would be nice to ignore this folder from the sonar scan - do you know if this is possible?

**Review by pinzart90 — APPROVED** [review]
LGTM

File: license_output/about-box_direct.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<div>
```
Comment (pinzart90) [inline license_output/about-box_direct.html:1]: not sure where you got this file from...

Comment (mjkkirschner) [reply license_output/about-box_direct.html:1]: the npm puller generates these files in the license_output folder.

**Pull request merged**